extensions/amp-carousel/0.1/base-slides.js

```diff
@@ -109,6 +114,31 @@ export class BaseSlides extends BaseCarousel {
     return false;
   }
 
+  /**
+  * Adds the pause button to the carousel
+  * @private
+  */
+  setupAutoplayButton_() {
```

Comment:
@nainar I think we should simply expose two `Actions` (https://github.com/ampproject/amphtml/blob/master/spec/amp-actions-and-events.md) and allow page authors to implement the UI however they like.